web/impact/impact/graphql/types/startup_type.py

```diff
@@ -1,9 +1,44 @@
+import graphene
 from graphene_django import DjangoObjectType
 
-from accelerator.models.startup import Startup
+from accelerator.models import (
+    Startup,
+    StartupStatus,
+)
+from impact.graphql.types import (
+    ProgramType
+)
 
 
 class StartupType(DjangoObjectType):
+    name = graphene.String()
+    high_resolution_logo = graphene.String()
+    program = graphene.List(ProgramType)
+
     class Meta:
         model = Startup
-        only_fields = ('short_pitch',)
+        only_fields = (
+            'id',
+            'short_pitch',
+        )
+
+    def resolve_name(self, info, **kwargs):
+        return self.name
+
+    def resolve_high_resolution_logo(self, info, **kwargs):
+        if self.high_resolution_logo:
+            return self.high_resolution_logo.url
+
+        return None
+
+    def resolve_program(self, info, **kwargs):
+
+        status = StartupStatus.objects.filter(
```

Comment:
If a startup has been a finalist in more than one program, would this get the right program?

Reply:
ordered this from newest to oldest based on creation_date, now for sure it will get the most recent startup status